hadoop-hdds/docs/content/feature/Reconfigurability.md

```diff
@@ -38,13 +38,18 @@ The meaning of command options:
     - **status**:     Check reconfig status
     - **properties**: List reconfigurable properties
 
-## OM Reconfigurability
+## Retrieve the reconfigurable properties list
+To retrieve all the reconfigurable properties list for a specific component in Ozone,
+you can use the command: `ozone admin reconfig properties --address=<ip:port>`.
```

Comment:
`ozone admin reconfig properties --address=<ip:port>` doesn't work.  The accepted format is what you used in the below examples.

Reply:
Already corrected